.gitignore

```diff
@@ -0,0 +1,3 @@
+# .gitignore
+node_modules/
+text.md 
```

Comment:
- [ ] [ OPTIONAL ] Because the previous reviewer did not mention this missed requirement, we close an eye and mark it as optional  🙃, you have well added the test.md file in the `.gitignore` file but you remove now the test.md file from the repo because you pushed it already by running this command `git rm file_name`

<img width="227" alt="1" src="https://user-images.githubusercontent.com/77843050/212075259-82c0ac09-057f-46eb-b63a-d1ee4616b0a7.png">

Reply:
Thank you for finding out my mistake. I will remove text.md as suggested .